sys/sys/efi.h

```diff
@@ -40,7 +40,7 @@
 #define	EFI_TABLE_SAL				\
 	{0xeb9d2d32,0x2d88,0x11d3,0x9a,0x16,{0x00,0x90,0x27,0x3f,0xc1,0x4d}}
 #define	EFI_TABLE_ESRT					\
-	{0xb122a263,0x3661,0x4f68,0x99,0x29,{0x78,0xf8,0xb0,0xd6,0x21,0x80}}
+	{0xb122a263,0x3661,0x4f68,{0x99,0x29,0x78,0xf8,0xb0,0xd6,0x21,0x80}}
```

Comment:
This change and `#undef`s above is why I was talking about header under `stand/`. They probably don't include `sys/` headers on purpose here. ioctl for ESRT likely won't work with this initializer because of `uuid_t`.

Maybe we should just add https://github.com/tianocore/edk2/blob/master/MdePkg/Include/Guid/SystemResourceTable.h to `sys/contrib/edk2` and include it? I suspect that might be the solution upstream would go for.

Reply:
@SergiiDmytruk I tried with that, but there are a lot of errors with unknown types like `UINT32` and `UINT64`. We should define these macros or include more from edk2, but I'm not sure that really is a more "clean" solution than that undef. Maybe it is enough for this time, and I should ask the community on the ML (at the upstream phase) which solution will be acceptable?

> ioctl for ESRT likely won't work with this initializer because of uuid_t
Ok, but ioctl is in the scope of the next task. It will be in different PR and I will still work on that

Reply:
> Maybe it is enough for this time, and I should ask the community on the ML (at the upstream phase) which solution will be acceptable?

OK, I'm fine with that.

Reply:
@SergiiDmytruk We have one discission left here - can you resolve that?

Reply:
Resolved.